Add CORS headers to fix issue with webfonts

panuta opened the pull request:
For web browser to be able to load and apply webfonts to web page correctly from different domain, we need to add these Cross-origin Resource Sharing headers to static files served from devtools

samuelcolvin commented on aiohttp_devtools/runserver/serve.py:
```diff
@@ -350,6 +350,10 @@ def _insert_footer(self, response):
             status, length = response.status, response.content_length
         else:
+            # Inject CORS headers to allow webfonts to load correctly
+            response.headers['Access-Control-Allow-Methods'] = 'GET'
```
Do we need `Access-Control-Allow-Methods`? If so should we allow more methods?

samuelcolvin replied:
I believe this isn't required.

samuelcolvin commented:
I think setting `Access-Control-Allow-Origin` for all files types/extensions is fine.

samuelcolvin commented:
Needs #165 to be merged before this will pass.

samuelcolvin commented:
please rebase or merge to current master.

panuta commented:
@samuelcolvin 

- I've removed http method check and added a test for this header.
- I've also rebase with latest commit from master and squash all my commits into one.

I'm not sure why Travis CI failed, though.

asvetlov commented:
Reuse `aiohttp-cors` maybe?

samuelcolvin commented:
@panuta looks good, but can you add a tests to confirm that `Access-Control-Allow-Origin` is present.

asvetlov commented:
One extra dependency is not a problem, aiohttp-devtools already depends on many libs. `pip` does it work perfectly fine.
I like to encourage good practices. Proper CORS support is not hard task for non-trivial cases, teaching library readers to `aiohttp-cors` is good thing I believe.

But it's very minor issue, I agree with any your decision -- you are aiohttp-devtools author :)

panuta commented:
@samuelcolvin I've added the test.

samuelcolvin commented:
Great thanks very much. I'll wait a couple of days to make sure we don't get any errors on v0.7 then release a patch with this in.
